tests/src/JIT/Regression/JitBlue/DevDiv_278523/DevDiv_278523.il

```diff
@@ -99,7 +76,9 @@
         br.s success
 
 _64bit:
-        call int32 C::Test64Bit(int32)
+        // This path can be reached when run on 64bits target, but not amd64.
+        // For example on arm64. There is no way to distinguish platform bitness in proj file.
```

Comment:
Is this the behavior we want on arm64, or do we actually want to be able to run the 64-bit version of this test for arm64?

Reply:
It is a good question.
Reasons why I think we should not run 64-bit version on arm64:
1. 64 test has incorrect IL, we want to support it on am64, because of backward compatibility. There are no reasons to support it on arm64.
2. We can't do the IL project condition to distinguish between 64 and 32 platforms in a general case. So we will have to write condition for each platform. If the new msbuild platform appears we will have to change the project file.

Reply:
> we want to support it on am64, because of backward compatibility. There are no reasons to support it on arm64.

I believe that, if we are supporting it on amd64, we should also support it on arm64, as people will expect even target-ptr-size-dependent code to work compatibly. I'm surprised that we can't specify bit-ness in the project file, but I think that it is still preferable even if we have to add the condition for each new platform.

Reply:
Ok, I will spend more time to find a way how to specify bit-ness.

Reply:
Fixed